projects/demo/src/modules/experimental/comment/examples/1/index.less

```diff
@@ -0,0 +1,3 @@
+:host {
+    width: 15rem;
```

Comment:
Could we drop it ?

<img width="1491" alt="comment-width" src="https://github.com/taiga-family/taiga-ui/assets/35179038/4e71d40b-7671-482e-b8e6-df3044f681d0">